src/config.rs

```diff
@@ -249,14 +239,10 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
 impl Config {
     /// Create a new configuration for creating Verso instance.
     pub fn new() -> Self {
+        servo_config::prefs::set(Preferences::default());
```

Comment:
We need to enable flexbox

Reply:
I believe it is currently enabled by default:

https://github.com/servo/servo/blob/47058816926d77a01151974cc793a4f2a7bda239/components/config/prefs.rs#L360